app/notification.go

```diff
@@ -164,6 +164,16 @@ func SendNotifications(post *model.Post, team *model.Team, channel *model.Channe
 				}
 			}
 
+			//If email verification is required and user email is not verified don't send email.
+			emailVerified := true
+			l4g.Debug("Server requires email verification: %v", utils.Cfg.EmailSettings.RequireEmailVerification)
```

Comment:
You'll want to remove these debug statements

Reply:
Complete